extensions/applicationinsights-angularplugin-js/projects/applicationinsights-angularplugin-js/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@microsoft/applicationinsights-angularplugin-js",
+  "version": "0.0.1",
```

Comment:
@MSNev this version number will be the package version number, should this match with other plugins as "2.5.X" or start from "0.0.1"? I agree with splitting up the plugins to its own repo 👍